pkg/helpers/client.go

```diff
@@ -116,10 +116,12 @@ func getClusterInfoKubeConfig(kubeClient kubernetes.Interface) (*clientcmdapiv1.
 }
 
 //WaitCRDToBeReady waits if a crd is ready
-func WaitCRDToBeReady(apiExtensionsClient apiextensionsclient.Interface, name string, b wait.Backoff) error {
+func WaitCRDToBeReady(apiExtensionsClient apiextensionsclient.Interface, name string, b wait.Backoff, wait bool) error {
 	errGet := retry.OnError(b, func(err error) bool {
 		if err != nil {
-			fmt.Printf("Wait  for %s crd to be ready\n", name)
+			if wait {
+				fmt.Printf("Wait  for %s crd to be ready\n", name)
+			}
```

Comment:
So this enable the progress message only if the wait option is true. But if the wait option is false, why we we waiting for the CRD to be ready?

Reply:
we should always wait for CRD ready to avoid [Race Condition](https://github.com/open-cluster-management-io/clusteradm/issues/47), so we kept the function name Wait in function name.
but I think the output should not be print to stdout while `--output json` is set, so I passed wait flag to it to check whether print the waiting status.